projects/plugins/jetpack/class.jetpack.php

```diff
@@ -706,7 +706,6 @@ private function __construct() {
 
 		add_filter( 'plugins_url', array( 'Jetpack', 'maybe_min_asset' ), 1, 3 );
 		add_action( 'style_loader_src', array( 'Jetpack', 'set_suffix_on_min' ), 10, 2 );
-		add_filter( 'style_loader_tag', array( 'Jetpack', 'maybe_inline_style' ), 10, 2 );
```

Comment:
Do we want to remove this filter altogether, or update it to use `wp_maybe_inline_style`? I'm truthfully not sure. Dashboard seems to render fine either way.

Reply:
I would say no. The wp_maybe_inline_style is loaded by default and used differently, not called directly. It's used by adding a path to style data:
```
wp_style_add_data( 'jetpack_facebook_likebox', 'path', plugin_dir_path( __FILE__ ) . 'facebook-likebox/style.css' );
```